lib/racecar/consumer.rb

```diff
@@ -63,7 +63,7 @@ def produce(topic:, payload:, key:, headers: nil)
     end
 
     def heartbeat
-      raise "not yet implemented for rdkafka"
+      warn "DEPRECATION WARNING: Manual heartbeats are not supported and not needed with librdkafka."
```

Comment:
In general 👍 
But we need to clarify with upstream how to handle breaking changes. I'd suggest to add those warnings in the current version and just remove this method.

Reply:
Agreed. I intentionally kept it here because it's easier to spot than the breaking change of removing it.